core/server/common/src/main/java/alluxio/cli/validation/UserLimitValidationTask.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.cli.validation;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+/**
+ * Task for validating system limit for current user.
+ */
+public final class UserLimitValidationTask implements ValidationTask {
+  private static final int NUMBER_OF_OPEN_FILES_MIN = 4096;
```

Comment:
Good spot on the doc. I think it's recommended to increase `maxproc` above 2048, at least to 16k in real production environment. Could you update the doc as well?